### PR TITLE
Enable rename option when copy or move a folder with a name already taken

### PR DESCRIPTION
**Pull request by gpuente**
## Description:

- Enable rename option when copy or move a folder with a name already taken

**NOTE:** this PR depends on https://github.com/powerhouse-inc/document-model-electron/pull/31 and https://github.com/powerhouse-inc/design-system/pull/113

### CODE REVIEW

File: src/hooks/useDrivesContainer.ts [hunk 1]
```diff
@@ -37,14 +36,14 @@ export function driveToBaseItems(
         id: driveID,
         label: drive.name,
         path: driveID,
-        type: ItemType.LocalDrive,
```
Comment (acaldas) [inline src/hooks/useDrivesContainer.ts:40]: We no longer export these types?

**Review by acaldas — COMMENTED** [review]
Really cool!

I'd prefer we keep using the ActionTypes and ItemTypes. Are we no longer exporting them in the design system?

File: src/hooks/useDrivesContainer.ts (hunk 1 above)
Comment (gpuente) [reply src/hooks/useDrivesContainer.ts:40]: nope, this is part of this refactor: https://github.com/powerhouse-inc/design-system/pull/111

**Comment by gpuente** [comment]
@acaldas I just pushed a new commit with the integration of the new types: [cfa7155](https://github.com/powerhouse-inc/document-model-electron/pull/33/commits/cfa7155eabf4ef28cb3dc8a87c8bb79656566907)

**Pull request merged**